RELEASE.md

```diff
@@ -7,6 +15,7 @@
 * Moved `_find_run_command()` and `_find_run_command_in_plugins()` from `__main__.py` in the project template to the framework itself.
 * Fixed a bug where `%load_node` breaks with multi-lines import statements.
 * Fixed a regression where `rich` mark up logs stop showing since 0.19.7.
+
```

Comment:
Can you add that we fixed the regression for the stacktrace not showing anymore?

Reply:
Done